src/WebSocket/ScoreWebSocket.qml

```diff
@@ -8,11 +8,10 @@ WebSocket {
     onTextMessageReceived: {
         try {
             // Print messages send by score
-            /*
```

Comment:
what you could do instead of decommenting / recommenting everytime, is to hide that behind a flag that e.g. reads an environment variable. For instance 

    if(g_debugMessagesEnabled) { console.log("bla"); }

with g_debugMessagesEnabled read with `qEnvironmentVariableIntValue("SCORE_QML_REMOTE_DEBUG") > 0` and passed to QML with e.g. https://doc.qt.io/qt-5/qtqml-cppintegration-contextproperties.html (likely `view.rootContext()->setContextProperty("g_debugMessagesEnabled", /* the value */);`

Then in the IDE you can set it on your side : https://doc.qt.io/qtcreator/creator-project-settings-environment.html

Reply:
but do that in another commit / branch / pr :)

Reply:
I will do that